DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -164,6 +165,21 @@ public void Get_course_content_of_unstarted_course_without_learning_should_retur
             result.Should().BeEquivalentTo(expectedCourse);
         }
 
+        [Test]
+        public void Get_course_content_for_removed_course_should_return_course()
+        {
+            // Given
+            const int candidateId = 210962;
+            const int customisationId = 24635;
+
+            // When
+            var result = courseContentService.GetCourseContent(candidateId, customisationId);
+
+            // Then
+            result.Should().NotBeNull();
+            result!.Sections.Where(section => section.PercentComplete != 0).Should().BeEmpty();

```

Comment:
I think comparing to an expected course content, like in the test above, will be a more thorough and easier to understand test 🙂